### ISSUE TITLE
Replace 'TTP build flag [bid:]' message with more meaningful one

### ISSUE THREAD

**Issue by jasdevsidhu12** [nishio-dens/bitbucket-pullrequest-builder-plugin#204]
How should I replace the message(picture below)'TTP build flag' to 'build fail' or 'build pass'
![image](https://user-images.githubusercontent.com/12964252/60661002-e3ec5780-9e50-11e9-8c76-9c16a8f84670.png)

**Comment by CodeMonk** [comment]
Sooo -- the build flag is not really saying pass or fail, it's saying that THAT particular commit has been built.

I'm pretty sure it gets added to the PR when the build STARTS, so that extra builds are not kicked off. The result (pass, fail, etc) is stored by atlassian somewhere in the build later.

But, I do agree that `TPP build flag` is awful. I'm happy to change that particular text to something more meaningful, like `BitbucketPullRequestBuilder: build:`

**Comment by jasdevsidhu12** [comment]
Thank you for the reply @CodeMonk , 

Yeah I would love to have the message "BitbucketPullRequestBuilder: build:" rather than "TTP build flag". 

I just wish there was setting in the PR plugin to display the message back to the user in BitBucket PR comment section whether the build pass or fail.

**Comment by CodeMonk** [comment]
Yeah, you can make the builder approve the build, but there isn't really an option to add a comment.

**Comment by CodeMonk** [comment]
Tell ya what. I'm going to re-open this one to have the build flag configurable-ish, or at least change it.

Please open another issue about commenting with build results - I think that would be a nice enhancement.

**Comment by ncarrier** [comment]
Hello,
Would it be possible to have a direct link to the actual jobs build in the comment?
That would allow to quickly check why a PR build failed.

And BTW, the hash should be that of the commit being built IIUC?
That's weird, I can't find this hash in my project's history...
